Cargo.toml

```diff
@@ -11,10 +11,10 @@ keywords = ["heap", "stash", "min-heap", "heap-sort", "collection"]
 categories = ["data-structures"]
 
 [dependencies]
-stash = "^0.1.3"
-itertools = "^0.5.9"
-unreachable = "^0.1.1"
-rand = "0.3"
+stash = "0.1.4"
+itertools = "0.9.0"
+unreachable = "1.0.0"
+rand = "0.5"
```

Comment:
Silly question but: Does the crate still compile and work and test passes with those updates?

Reply:
I checked the following scripts work:

```sh
cargo test
cargo +nightly bench --features bench
```

Reply:
Okay in this case I trust you and the PR should be fine.